src/heretic/g_game.c

```diff
@@ -1663,12 +1737,40 @@ void G_RecordDemo(skill_t skill, int numplayers, int episode, int map,
                   char *name)
 {
     int i;
+    int maxsize;
+
+    // If not recording a longtics demo, record in low res
+
+    lowres_turn = !longtics;
+
+    //!
+    // @category demo
+    //
+    // Smooths out low turning resolution when recording a demo.
```

Comment:
This doc string will appear in manpages. Please rephrase this to imperative mood, ie.

"Smooth out low resolution turning when recording a demo."